post/views.py

```
@@ -0,0 +1,89 @@
+        post.delete()
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+    @swagger_auto_schema(
+        operation_id="게시글 수정",
+        operation_description="게시글을 수정합니다.",
+        responses={200: "PostSerializer", 404: "Not Found", 400: "Bad Request"},
+    )
+    def patch(self, request, post_id):
+        try:
+            post = Post.objects.get(id=post_id)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)